bibliopixel/layout/circle.py

```diff
@@ -59,17 +59,19 @@ def layout_from_rings(rings, origin=(0, 0, 0), z_diff=0):
 
     points = [None] * num
     z = 0
+
     for i in range(len(rings)):
         r = rings[i]
         step = steps[i]
         angle = 0.0
         for p in r:
-            radius = (len(r) * 0.5)
+            radius = (len(r) - 1) * 3.14
```

Comment:
Use `math.pi`! 

https://en.wikipedia.org/wiki/Indiana_Pi_Bill

Otherwise, looks good to me.